modules/cnf-configuring-cvl-nic-as-oc.adoc

```diff
@@ -0,0 +1,206 @@
+// Module included in the following assemblies:
+//
+// * networking/using-ptp.adoc
+
+:_content-type: PROCEDURE
+[id="cnf-configuring-cvl-nic-as-ptp-slave_{context}"]
+= Configuring linuxptp services as ordinary clock for Intel Columbiaville NIC
+
+You can configure `linuxptp` services as a single ordinary clock for nodes with an Intel 800-Series Columbiaville NIC installed.
+
+[NOTE]
+====
+The below `PtpConfig` CR configures PTP fast events by setting values for `ptp4lOpts`, `ptp4lConf` and `ptpClockThreshold`.
```

Comment:
```suggestion
The `PtpConfig` CR configures PTP fast events by setting values for `ptp4lOpts`, `ptp4lConf` and `ptpClockThreshold`.
```

I don't think below is required, but ignore if it changes the meaning of the sentence.